slsim/lens.py

```diff
@@ -355,7 +355,7 @@ def deflector_redshift(self):
 
         :return: lens redshift
         """
-        return self.deflector.redshift
+        return float(self.deflector.redshift)
```

Comment:
I already made changes in my PR for this. I'd like to have the self.deflector.redshift quantity well defined as float, not the return of the function in lens.py (same for the source redshifts as well